src/Main.java

```diff
@@ -1,57 +1,25 @@
+import Task2.Bulldozer;
+import Task2.Machine;
+import Task2.Tractor;
+import Task2.Truck;
+
 import java.util.ArrayList;
-import java.util.Arrays;
+
 
 public class Main {
     public static void main(String[] args) {
-        String s = "ver12 nika 43 2 !23 0";
-        System.out.println(Arrays.toString(numberFromString(s)));
-        System.out.println(Arrays.toString(numberFromString("ver12 nika 43 2 !23 0 qenfj &iqef483 wwir2")));
-        System.out.println(Arrays.toString(numberFromString("dnjfbn3431 rignwrin>sfjgj21wgr /e4in44oewfm")));
-
-    }
-
-    static int[] numberFromString(String str){
-        String numStr = "";
-        ArrayList<Integer> arr = new ArrayList<>();
-
-        for (int i = str.length()-1; i >= 0; i--){
-
-            if (Character.isDigit(str.charAt(i))){
-                numStr = str.charAt(i) + numStr;
+        ArrayList<Machine> arr = new ArrayList<>();
```

Comment:
We operate on interfaces level not classes

Reply:
Okay, but I don't understand

Reply:
What is the purpose of inheritance? Can I place a begin and end methods to the main class?

Reply:
The purpose of inheritance is to take all characteristics of one class and use them in another. I think yes